doc/man3/OSSL_PARAM.pod

```diff
@@ -259,6 +263,10 @@ must fill all I<data_size> bytes and ensure correct padding for the
 native endianness, and set I<return_size> to the same value as
 I<data_size>.
 
+=item *
+
+The array must be terminated by B<OSSL_PARAM_END>.
```

Comment:
Change `by B<...>` to:
```
with:
 { NULL, 0, NULL, 0, NULL }
```

Reply:
While at it, correct `{ NULL, 0, NULL, 0, NULL }` to `{ NULL, 0, NULL, 0, 0 }`

Reply:
Yeah, okay.

I would consider this change to be trivial.

Reply:
If you are doing an initializer, just use ```{NULL}``` and let the C language fill in all the other fields.